vpr/src/base/vpr_context.h

```diff
@@ -573,33 +573,16 @@ struct NocContext : public Context {
  * This should contain only data structures that
  * related to server state.
```

Comment:
related -> relate to the vpr server state.

Reply:
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/ecd73778c94aa19f60d928b09c6818d3242c66df
(will be updated later)